packages/flutter/test/animation/animations_test.dart

```diff
@@ -240,14 +238,14 @@ void main() {
       vsync: const TestVSync(),
     );
     final CurvedAnimation curved = CurvedAnimation(parent: controller, curve: BogusCurve());
-    FlutterError error;
+    FlutterError? error;
```

Comment:
This can be `late`.

Reply:
Discussed in https://github.com/flutter/flutter/pull/62616#discussion_r499060405